asyn ships the generated stubs in asyn/vxi11/rpc/, but regenerates them with
rpcgen everywhere except vxWorks and RTEMS. rpcgen is not a conda package, so
we make the mechanism apply to all platforms.

--- a/asyn/Makefile
+++ b/asyn/Makefile
@@ -324,7 +324,7 @@ drvVxi11$(DEP): vxi11intr.h vxi11core.h
 vxi11core_xdr.c$(DEP): vxi11core.h
 
 ifdef T_A
- ifneq ($(findstring $(OS_CLASS),vxWorks RTEMS),)
+ ifeq (1,1)
 
 vxi11core.h vxi11core_xdr.c vxi11intr.h vxi11intr_xdr.c : \
 % : ../vxi11/rpc/%
